builder/builder.go

```diff
@@ -259,13 +260,19 @@ func (b *Builder) checkAndRunPreReqs(runnable context.RunnableDir, result *Build
 	}
 
 	for _, p := range preReqs {
+
 		filepath := filepath.Join(runnable.Fullpath, p.File)
 
 		if _, err := os.Stat(filepath); err != nil {
 			if errors.Is(err, os.ErrNotExist) {
 				b.log.LogStart(fmt.Sprintf("missing %s, fixing...", p.File))
 
-				outputLog, err := util.RunInDir(p.Command, runnable.Fullpath)
+				fullCmd, err := p.GetCommand(runnable)
+				if err != nil {
+					return errors.Wrap(err, "failed to get templated full command")
```

Comment:
It maybe seems a little silly, but the convention we use for error messages is `failed to` + the name of the function that failed. Something like this:
```suggestion
					return errors.Wrap(err, "failed to GetCommand for preReq")
```

Reply:
I'll change that up. Also somewhat relatedly, what do you think of using this? https://thanos.io/tip/contributing/coding-style-guide.md/#wrap-errors-for-more-context-dont-repeat-failed--there

> NOTE: never prefix wrap messages with wording like failed ... or error occurred while.... Just describe what we wanted to do when the failure occurred. Those prefixes are just noise. We are wrapping error, so it’s obvious that some error occurred, right? (: Improve readability and consider avoiding those.

Reply:
I changed up the errors in [64cb16d](https://github.com/suborbital/subo/pull/187/commits/64cb16de97cd55150339a1e381794ed04c531a91).

Additionally I see that `result.OutputLog` gets the log from `util.RunInDir` but it's not being used anywhere that I can see. A global search doesn't reveal anything where `OutputLog` gets passed to a log function call, or even to an fmt.

Reply:
@javorszky I'm a fan of that honestly. Will ping @cohix for an opinion as well since he came up with the format that's used currently.